packages/core/src/js/feedback/integration.ts

```diff
@@ -42,3 +54,36 @@ export const getFeedbackButtonOptions = (): Partial<FeedbackButtonProps> => {
 
   return integration.buttonOptions;
 };
+
+export const getColorScheme = (): 'system' | 'light' | 'dark' => {
+  const integration = getClient()?.getIntegrationByName<ReturnType<typeof feedbackIntegration>>(
+    MOBILE_FEEDBACK_INTEGRATION_NAME,
+  );
+  if (!integration) {
+    return 'system';
+  }
+
+  return integration.colorScheme;
+};
+
+export const getFeedbackLightTheme = (): Partial<FeedbackWidgetTheme> => {
+  const integration = getClient()?.getIntegrationByName<ReturnType<typeof feedbackIntegration>>(
+    MOBILE_FEEDBACK_INTEGRATION_NAME,
+  );
+  if (!integration) {
+    return {};
+  }
```

Comment:
The helper functions look good, but since we are using them more and more, let's extract the core (integration getter) to a helper to avoid repeating it.

Reply:
Good idea 👍 
Extracted a helper function with 6e7b61ba78de9f3cebc6056f7dc2f826fd4b5500